typescript/sdk/src/token/xerc20.ts

```diff
@@ -149,6 +149,7 @@ async function getConfigurationChangedLogsFromRpc({
   const currentBlockNumber = await provider.getBlockNumber();
   return provider.getLogs({
     address: xERC20Address,
+    fromBlock: 0,
```

Comment:
I'm actually not sure you can do this, because some or maybe even most rpc providers don't let you query for more than 2000 blocks

Reply:
You are right, but filtering by topic without setting the block boundaries should be enough in this case, I'll test with some public rpc to see if it works well

Reply:
update: seems to work fine just by setting the upper block number and filtering by topic